osu.Game/Screens/Edit/Timing/TapTimingControl.cs

```diff
@@ -179,6 +187,9 @@ private void load(OverlayColourProvider colourProvider, OsuColour colours)
                 if (handling.NewValue)
                     start();
             }, true);
+
+            track.BindTo(clock.Track);
+            track.ValueChanged += _ => waveformContainer.Child = new WaveformComparisonDisplay();
```

Comment:
A bit dangerous this. If the track happens to change for whatever reason during load then this'll throw because of invalid thread for mutation. Probably best to move to `LoadComplete()` for safety.